src/modules/ROOT/pages/changelog/202303/03.json

```diff
@@ -0,0 +1,93 @@
+{
+  "meta": {
+    "releasedOn": "2023.06.27"
+  },
+  "added": [
+    {
+      "what": [
+        "a warning message that notifies the user if they try to leave a page with unsaved changes on it."
+      ],
+      "why": [
+        "to prevent loosing changes that have't been saved yet."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1051"
+      ]
+    },
+    {
+      "what": [
+        "the option to add custom styles to all of the web applications. The styles can be added via the user settings in {APP_NAME_ADMIN}."
+      ],
+      "why": [
+        "to provide the user with more freedom to style the applications as they see fit."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1262"
+      ]
+    }
+  ],
+  "changed": [
+    {
+      "what": [
+        "the provided parameter in the url of a page for selecting one or more records in a record list in {APP_NAME_COMPOSE}."
+      ],
+      "why": [
+        "previously a value wasn't provided, whereas now it is."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza/pull/1244"
+      ]
+    },
```

Comment:
I'm not sure I get this. Also, if it wasn't there before, is this an addition?

Reply:
CL not needed for this